.travis.yml

```diff
@@ -4,4 +4,7 @@ addons:
 node_js:
   - "6"
   - "8"
-script: yarn lint && yarn test && yarn add --dev webpack@3 && yarn test
+env:
+  - WEBPACK_VERSION=4
+  - WEBPACK_VERSION=3
+script: yarn lint && yarn add --dev webpack@$WEBPACK_VERSION && yarn test
```

Comment:
you may want to override “install” for the add command; and leave script to only be yarn test,

Also, you may want to run yarn lint in its own build, rather than running it 4 times

Reply:
True, though linting is so quick in this project that it doesn't matter much.